src/components/structures/RoomView.js

```diff
@@ -33,7 +32,7 @@ import { _t } from '../../languageHandler';
 import {RoomPermalinkCreator} from "../../matrix-to";
 
 const MatrixClientPeg = require("../../MatrixClientPeg");
-const ContentMessages = require("../../ContentMessages");
+import ContentMessages from '../../ContentMessages';
```

Comment:
extreme nit: double quotes for consistency with the file

Reply:
TBH the requires / imports and quotes were both all over the place here so I've just made them all comply with the code style rather than makes this one not comply for consistency with the rest.